Runtime/Scripts/SceneExporter/ExporterMaterials.cs

```diff
@@ -909,6 +951,10 @@ private void ExportPBRSpecularGlossiness(GLTFMaterial material, Material materia
 			{
 				diffuseFactor = materialObj.GetColor("_Color").ToNumericsColorLinear();
 			}
+			else if (materialObj.HasProperty("_Tint"))
+			{
+				diffuseFactor = materialObj.GetColor("_Tint").ToNumericsColorLinear();
+			}
```

Comment:
I dont think tint should just overwrite the diffuse color. If there is no separate value for it in the gltf spec, which looks to be the case (https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#metallic-roughness-material) then it should be _multiplied_ in, not overwrite

Reply:
Also can`t _Tint and _BaseColor be there simultaneously?